### PR TITLE
#3971 update release notes for 2.4

### PR DESCRIPTION
**Pull request by planetf1**
Signed-off-by: Nigel Jones <nigel.l.jones+git@gmail.com>

Updates release notes for 2.4

Please review/comment & suggest any updates or push to this PR branch.

I would like to get an update out tomorrow since the build is now publishing. However we can make further incremental updates after that as the release notes are in master.

Reviewers have added 1 or more commits in this release.

### CODE REVIEW

**Comment by sarbull** [comment]
We should also mention the UI extraction.

**Comment by planetf1** [comment]
> We should also mention the UI extraction.

Agreed - thanks for spotting

**Comment by planetf1** [comment]
@sarbull - I've added a small section on the static UI. Any good?

**Comment by grahamwallis** [comment]
I wonder whether we should say "The Repository explorer User Interface ('REX') must now be configured with a list of available platforms and servers..." rather than saying that it "can" be. We could add that this is to improve usability and security.

File: release-notes/release-notes-2-4.md [hunk 1]
```diff
@@ -1,29 +1,33 @@
...
 
-# Release 2.4 (Planned October 2020)
+# Release 2.4 (October 2020)
 
-Release 2.4 adds support for:
-   * Project management of IT change
-   * IoT and large scale cohorts
+* The [Integration Daemon](https://github.com/odpi/egeria/blob/master/open-metadata-implementation/admin-services/docs/concepts/integration-daemon.md) now makes is simpler to exchange metadata with third party technology 
+  such as catalogs and databases.
+* The Repository explorer User Interface ('REX') can now be configured with a list of available
```
Comment (grahamwallis) [inline release-notes/release-notes-2-4.md:8]: Need to be explicit that this is the presentation-server implementation of Rex.

File: release-notes/release-notes-2-4.md [hunk 2]
```diff
@@ -1,29 +1,33 @@
...
+  such as catalogs and databases.
+* The Repository explorer User Interface ('REX') can now be configured with a list of available 
+  platforms and servers to improve security and usability. 
+* The latest version of Repository Explorer, Type Explorer and Dino, is found 
+  in the 'Presentation Server' UI. See 
+  [Presentation Server component documentation](https://github.com/odpi/egeria/tree/master/open-metadata-implementation/user-interfaces/presentation-server) 
+  and 
+  [Configuring the Presentation Server](https://github.com/odpi/egeria/blob/master/open-metadata-implementation/admin-services/docs/user/configuring-the-presentation-server.md)
+  .
+* The [Business UI ](https://github.com/odpi/egeria/tree/master/open-metadata-implementation/user-interfaces/ui-chassis/ui-chassis-spring)
```
Comment (davidradl) [inline release-notes/release-notes-2-4.md:16]: I do not think Business UI is the right term for this . The Glossary Author is a "Business UI" in the Presentation server. I suggest Web Component UI

**Review by davidradl — CHANGES_REQUESTED** [review]

File: release-notes/release-notes-2-4.md (hunk 1 above)
Comment (planetf1) [reply release-notes/release-notes-2-4.md:8]: agreed - switched it around a bit (and can->must)

File: release-notes/release-notes-2-4.md (hunk 2 above)
Comment (planetf1) [reply release-notes/release-notes-2-4.md]: I'm going to counter propose staying with the 'UI Chassis'. I agree Business UI is confusing for end user, but Web Component UI could be either - what is it.

We need to clarify the role of the UIs, but that's a longer task. For now I think staying with UI chassis makes sense, as that at least is already used, linked, and referred to in the broader docs & code.

**Comment by planetf1** [comment]
I plan to merge this once built. If we have followup updates please raise a new issue.
The release artifacts are going out so I want to ensure anyone hitting the link sees reasonable information even if imperfect.

**Pull request merged**